sdv/tabular/ctgan.py

```diff
@@ -79,6 +79,15 @@ def _sample(self, num_rows, conditions=None):
 
         raise NotImplementedError(f"{self._MODEL_CLASS} doesn't support conditional sampling.")
 
+    def _set_random_state(self, random_state):
+        """Set the random state of the model's random number generator.
+
+        Args:
+            random_state (int, np.random.RandomState, or None):
```

Comment:
Wouldn't this be `tuple[np.random.RandomState, torch.Generator]` instead of just `np.random.RandomState`?